src/test/java/com/bieganski/ox/PositionCalculatorTest.java

```diff
@@ -0,0 +1,18 @@
+package com.bieganski.ox;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class PositionCalculatorTest {
```

Comment:
I would add more test here. This is quite important class and there are many alternative scenarios to test.